package.json

```
@@ -18,7 +18,7 @@
     "chroma-js": "2.1.0",
     "gatsby": "2.19.32",
     "gatsby-plugin-fathom": "1.2.0",
-    "gatsby-theme-austere": "0.1.0",
+    "gatsby-theme-austere": "0.1.1",
```

```suggestion
    "gatsby-theme-austere": "0.1.0",
```

We can move this back to 0.1.0 and then you can run `yarn changeset` locally to automatically make a new patch for release.

---

The version was changed to `0.1.1` in https://github.com/johno/johno.com/commit/c9c4d46c71d8af4ec4802b3b74c18ea904c88cc1

But it wasn't update in `package.json`, which means when I tried to run `yarn` it failed.